assets/queries/terraform/aws/s3_bucket_with_unsecured_cors_rule/query.rego

```diff
@@ -14,5 +14,22 @@ CxPolicy[result] {
 		"issueType": "IncorrectValue",
 		"keyExpectedValue": "'cors_rule' does not allows all methods, all headers or several origins",
 		"keyActualValue": "'cors_rule' allows all methods, all headers or several origins",
+		"searchLine": common_lib.build_search_line(["resource", "aws_s3_bucket", name, "cors_rule"], []),
+	}
+}
+
+CxPolicy[result] {
+	module := input.document[i].module[name]
+	keyToCheck := common_lib.get_module_equivalent_key("aws", module.source, "aws_s3_bucket", "cors_rule")
+	rule := module.cors_rule
```

Comment:
I think here should be an array check, since `cors_rule` is a list of maps, not a single map, as you can see [here](https://registry.terraform.io/modules/terraform-aws-modules/s3-bucket/aws/latest#input_cors_rule). This would look like:
```suggestion
	rule := module.cors_rule[ruleIdx]
```